production-dependency-templates/exocom/README.md

```diff
@@ -0,0 +1,53 @@
+# Exocom dependency
+
+_Production plugin for RDS-type dependencies_
+
+## Usage
+- List database `exocom` as a remote dependency in `application.yml`, with `type: exocom`
```

Comment:
I believe database should be removed.

Reply:
This feels like general configuration for any production dependency and should probably be in a top level readme since it applies to all versions. I think we could just have sections that specify the type and one that specifies the template config.